.github/workflows/test.yml

```diff
@@ -8,17 +8,17 @@ jobs:
     strategy:
       matrix:
         test-platform: [vm, chrome]
-        dart-channel: ["2.12.0", "2.13.0"]
+        dart-channel: ["3.4.4", "3.4.4"]
```

Comment:
i was thinking that it could be cool to have both limits, the minimum version and the maximum version so it validates that it compiles on the range, WDYT?